packages/react-mutation-mapper/src/component/lollipopPlot/LollipopPlotNoTooltip.tsx

```diff
@@ -689,11 +689,12 @@ export default class LollipopPlotNoTooltip extends React.Component<
         groupName?: string,
         symbol: string = '#'
     ) {
-        const label = groupName
+        const label = this.props.yAxisLabelFormatter
+            ? this.props.yAxisLabelFormatter(symbol, groupName)
+            : groupName
```

Comment:
We should avoid nested ternary conditionals. Let's add an if else statement here for `this.props.yAxisLabelFormatter`